backend/src/main/java/de/jankahle/capstone/model/Ingredient.java

```diff
@@ -0,0 +1,16 @@
+package de.jankahle.capstone.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Ingredient {
+    private String name;
+    private int amount;
```

Comment:
depending on where amount is later used it might cause problems - may want to consider a float e.g. for 3/4 or a string as well

Reply:
good point, I'll adjust that